dist/extensions/CirrusSearch/maintenance/UpdateOneSearchIndexConfig.php

```diff
@@ -523,6 +576,21 @@ public function getIndex() {
 			$this->indexBaseName, $this->indexSuffix, $this->indexIdentifier );
 	}
 
+	/**
+	 * @return \Elastica\Index being updated
+	 */
+	public function getSharedIndex() {
+		return $this->getConnection()->getIndex(
+			$this->indexSharedName, $this->indexSuffix, $this->indexIdentifier );
+	}
+
+	/**
+	 * @return \Elastica\Index being updated
+	 */
+	public function getRealIndex() {
```

Comment:
suggestion: this can probably be a private method (I really hope we never see a reason to call this from anywhere else)

suggestion: should this have a name other than "real"? perhaps "getNotAliasedIndex"?